Header header
geometry_msgs/PoseWithCovariance pose
geometry_msgs/Pose pose_corrections
float64 outlier_percentage
float64 aligment_fitness